internal/processor/parser/app.go

```diff
@@ -194,26 +194,25 @@ func listApps(c *cfclient.Client, numWorkers int, log *gosteno.Logger) ([]cfclie
 	pages := resp.Pages - 1
 	// No need for more workers than pages left
 	numWorkers = int(math.Min(float64(numWorkers), float64(pages)))
+	var pagesPerWorker int
```

Comment:
Shouldn't it be initialized to `0`?

Reply:
automatically done by go using this syntax